tests/conftest.py

```diff
@@ -32,6 +32,16 @@
 }
 
 
+@pytest.fixture(
+    params=[
+        pytest.param("asyncio", marks=pytest.mark.asyncio),
+        pytest.param("trio", marks=pytest.mark.trio),
+    ]
+)
+def async_environment(request) -> str:
```

Comment:
Should we have a small docstring here, noting usage, and that this is equivelent to having a pair of tests, wrapped wth `@pytest.mark.asyncio` and `@pytest.mark.trio`. Or perhaps not neccessary?

Reply:
Sounds good, yup